contracts/schemes/TransitionScheme.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity 0.5.17;
+
+import "../controller/Controller.sol";
+
+/**
+ * @title A scheme for transitioning the DAO's assets to a new one.
+ */
+
+contract TransitionScheme {
+
+    Avatar public avatar;
+    address payable public newAvatar;
+    address[] public externalTokens;
+    address[] public assetAddresses;
+    bytes4[] public selectors;
+
+    /**
+     * @dev initialize
+     * @param _avatar the avatar to migrate from
+     * @param _newAvatar the avatar to migrate to
+     * @param _externalTokens external tokens to allow transfer to the new avatar
+     * @param _assetAddresses the assets to transfer
+     * @param _selectors the functions to call to to transfer the assets
```

Comment:
Does this include reputation?

Reply:
No, reputation is recreated in the new DAO

Reply:
no

Reply:
Is there any other plan for transferring rep from a 1.0 DAO to a 2.0 DAO?

Reply:
sure.